harmony_netcdf_to_zarr/convert.py

```diff
@@ -363,8 +315,7 @@ def __copy_variable(netcdf_variable: NetCDFVariable, zarr_group: ZarrGroup,
                                                   aggregated_dimensions)
         new_chunks = compute_chunksize(aggregated_shape, dtype)
 
-        fill_value = scaled.get('_FillValue',
-                                getattr(netcdf_variable, '_FillValue', 0))
+        fill_value = getattr(netcdf_variable, '_FillValue', 0)
```

Comment:
The default value for `getattr` being 0 matches the default value for the `fill_value` keyword argument in `require_dataset` below.